pages/contribute.md

```diff
@@ -6,44 +6,58 @@ main_nav_id: contribute
 
 # Contributing to Kotlin
 
-Kotlin is an Apache 2 OSS Project. The source code, tooling, documentation and even this web site is maintained on [GitHub](https://github.com/jetbrains/kotlin). While the majority 
-of the team works at JetBrains, there have been nearly a hundred external contributors to the Kotlin Project and we are always looking for more people. 
+Kotlin is an Apache 2 OSS Project. The source code, tooling, documentation and even this web site is maintained on [GitHub](https://github.com/jetbrains/kotlin). 
+While Kotlin is mostly developed by JetBrains, there are hundreds of external contributors to the Kotlin project, and we always look for more people&mdash;[become one of them](https://kotlinlang.org/community). 
 
-## How you can contribute
+## Participate in Early Access Preview
 
-### Contributing to the code
+You can help us improve Kotlin if you participate in Kotlin Early Access Preview (EAP) and provide your valuable feedback.
+ 
+For every release, Kotlin ships a few preview builds where you can try out the latest features before they go to production. You can report found bugs in [YouTrack](https://kotl.in/issue) 
+and we will try to fix them in the next EAP build. This way, you can get bug fixes earlier than the standard Kotlin release cycle.
 
-If you want to contribute to the code, go to GitHub and check out the latest version and follow the instructions on how to build Kotlin from source. After that, you can start picking
-some [pending tasks](https://youtrack.jetbrains.com/issues/KT?q=%23Unresolved+) on the issue tracker. Make sure you look for issues tags with [Up For Grabs](https://youtrack.jetbrains.com/issues/KT?q=%23Unresolved+tag%3A+%7BUp+For+Grabs%7D) as
-these are some of the easier ones to get started with.
+## Contribute to the compiler, standard library, and tooling
 
-### Contributing to the documentation or web site
+If you want to contribute to the Kotlin compiler, standard library, or tooling, go to [GitHub](https://github.com/jetbrains/kotlin), check out the latest Kotlin version,
+ and follow the [instructions on how to contribute to the code](https://github.com/JetBrains/kotlin/blob/master/docs/contributing.md).
+ 
+You can help us by completing [open tasks](https://youtrack.jetbrains.com/issues/KT?q=tag:%20%7BUp%20For%20Grabs%7D%20and%20State:%20Open). 
+Please stay in touch with us because we may have questions and comments on your changes. Otherwise, we won't be able to incorporate them. 
```

Comment:
Please keep an open line of communication with us because we may have questions and comments on your changes. Otherwise, we won't be able to incorporate your contributions.

Reply:
done